src/pages/courseSettings.js

```diff
@@ -0,0 +1,269 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Form,
+  CustomInput,
+  Col,
+  FormText,
+  FormGroup,
+  Label,
+  Input,
+  Button,
+  Container,
+  Card,
+  CardHeader,
+  CardTitle,
+  ListGroup,
+  ListGroupItem,
+} from 'reactstrap'
+import { connect } from 'react-redux'
+import FlipMove from 'react-flip-move'
+
+import {
+  fetchCourse,
+  fetchCourseRequest,
+  addCourseStaff,
+  removeCourseStaff,
+  updateUnlistedCourse,
+  updateQuestionFeedback,
+} from '../actions/course'
+
+import Error from '../components/Error'
+import PageWithUser from '../components/PageWithUser'
+import AddStaff from '../components/AddStaff'
+import RemoveableUserItem from '../components/RemoveableUserItem'
+import { withBaseUrl } from '../util'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faDownload } from '@fortawesome/free-solid-svg-icons'
+import ShowForAdmin from '../components/ShowForAdmin'
+
+class CourseSettings extends React.Component {
```

Comment:
I'd prefer to structure this more like the `QueueSettings` component, where most logic is pushed down into separate panels components. This helps us avoid massive ~300 line components like this one 🙂 

In general, I'd try to mirror the `QueueSettings` component as much as possible in terms of design and functionality:
* No bright blue headers
* Explicit "Update" buttons
* Don't break each setting into a separate panel - group them under a "General" panel